bagit.py

```diff
@@ -501,7 +501,7 @@ def files_to_be_fetched(self):
     def has_oxum(self):
         return 'Payload-Oxum' in self.info
 
-    def validate(self, processes=1, fast=False):
+    def validate(self, processes=1, fast=False, completeness=False):
```

Comment:
Yeah, the name for this is tough, since I want to make sure it doesn't signal that all checks are running. I think `completeness_only` is a good addition here. Any thoughts about the CLI argument?